### PR TITLE
[0.13.x] KOGITO-2921 Bump to Quarkus 1.7.0.CR1

### PR DESCRIPTION
**Pull request by evacchi**
- https://github.com/kiegroup/kogito-runtimes/pull/671
- https://github.com/kiegroup/kogito-apps/pull/367
- https://github.com/kiegroup/kogito-examples/pull/318

### CODE REVIEW

**Comment by evacchi** [comment]
@cristianonicolai halp. Apparently some stuff under UI broke?

**Comment by evacchi** [comment]
I am trying to fold https://github.com/kiegroup/kogito-apps/pull/363 into this one I think they can't really stand alone

**Comment by evacchi** [comment]
~~so this is one error I am getting here~~ it is actually a warning

```
            "message": "Exception while fetching data (/UserTaskInstances) : org.hibernate.search.exception.SearchException: HSEARCH000301: Requested sort field(s) state are not configured for entity type org.infinispan.query.remote.impl.indexing.ProtobufValueWrapper mapped to index usertaskinstances_protobuf, thus an uninverting reader must be created. You should declare the missing sort fields using @SortableField.",
```

**Comment by cristianonicolai** [comment]
@evacchi please remove yarn.lock file

**Comment by cristianonicolai** [comment]
@evacchi I think we can skip the changes in the proto file for now, sohuldnt be needed as we're running with Infinispan 10 anyway.

File: pom.xml [hunk 1]
```diff
@@ -16,12 +16,15 @@
   <description>Kogito Apps</description>
 
   <properties>
-    <version.io.quarkus>1.6.0.Final</version.io.quarkus>
```
Comment (cristianonicolai) [inline pom.xml]: @evacchi actually, should we consider keeping the version here to avoid any unexpected changes like infinispan client etc... ?

Comment (evacchi) [reply pom.xml]: right

Comment (evacchi) [reply pom.xml:19]: don't we need to basically override every change in runtimes pom?

Comment (evacchi) [reply pom.xml]: don't we need to basically override every change in runtimes pom?

Comment (evacchi) [reply pom.xml]: I think that the runtimes pom is incorrectly exposing this property+dependency:

```
    <version.io.smallrye.reactive>2.2.1</version.io.smallrye.reactive>
      <dependency>
        <groupId>io.smallrye.reactive</groupId>
        <artifactId>smallrye-reactive-messaging-provider</artifactId>
        <version>${version.io.smallrye.reactive}</version>
      </dependency>
```

i'll try and fix that and see how it goes

Comment (cristianonicolai) [reply pom.xml]: > don't we need to basically override every change in runtimes pom?

yeah, think so

**Review by radtriste — APPROVED** [review]

File: data-index/data-index-storage/data-index-storage-protobuf/src/main/resources/META-INF/kogito-index.proto [hunk 2]
```diff
@@ -127,7 +127,9 @@ message UserTaskInstance {
     optional string rootProcessInstanceId = 7;
     /* @Field(store = Store.YES) */
     optional string rootProcessId = 8;
-    /* @Field(store = Store.YES) */
+    /* @Field(store = Store.YES)
```
Comment (cristianonicolai) [inline data-index/data-index-storage/data-index-storage-protobuf/src/main/resources/META-INF/kogito-index.proto]: @evacchi please revert the proto changes as they are not needed.

Comment (evacchi) [reply data-index/data-index-storage/data-index-storage-protobuf/src/main/resources/META-INF/kogito-index.proto]: ok

**Review by tiagodolphine — APPROVED** [review]

**Review by cristianonicolai — APPROVED** [review]

**Comment by evacchi** [comment]
jenkins please retest this

**Comment by evacchi** [comment]
jenkins please retest this

**Pull request merged**